crates/compiler/test_gen/src/gen_list.rs

```diff
@@ -3004,8 +3006,10 @@ fn list_find_empty_layout() {
             List.findFirst [] \_ -> True
             "#
         ),
-        RocResult::err(()),
-        RocResult<(), ()>
+        // [Ok [], Err [NotFound]] gets unwrapped all the way to just [NotFound],
+        // which is the unit!
+        (),
+        ()
```

Comment:
wow we really need to remember this. Easy to mess that up and wonder why you get no bytes

Reply:
`glue` should take care of this in the host case at least!